src/store/index.ts

```diff
@@ -1,51 +1,65 @@
 import { createStore, AnyAction } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import { TodoWithUser } from '../interfaces';
+import { sortBy } from './sortBy';
 
-// Action types - is just a constant. MUST have a unique value.
-const START_LOADING = 'START_LOADING';
-const FINISH_LOADING = 'FINISH_LOADING';
+const ACTIONS = {
+  START_LOADING: 'START_LOADING',
+  SET_TODOS: 'SET_TODOS',
+  SET_VISIBLE_TODOS: 'SET_VISIBLE_TODOS',
+  SET_ORDER: 'SET_ORDER',
+};
 
-// Action creators - a function returning an action object
-export const startLoading = () => ({ type: START_LOADING });
-export const finishLoading = (message = 'No message') => ({ type: FINISH_LOADING, message });
+export const setVisibleTodos = (field: string) => ({ type: ACTIONS.SET_VISIBLE_TODOS, payload: field });
+export const setTodos = (todos: TodoWithUser[]) => ({ type: ACTIONS.SET_TODOS, payload: todos });
+export const startLoading = () => ({ type: ACTIONS.START_LOADING });
```

Comment:
startLoading() // loading started
startLoading() // loading finished

![image](https://user-images.githubusercontent.com/36257947/88378860-8227c400-cdaa-11ea-9100-7a22e4eefb0a.png)
```suggestion
export const toggleLoading = () => ({ type: ACTIONS.TOGGLE_LOADING });
```